bin/fuel-convert

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+import argparse
+import importlib
+import pkgutil
+
+from fuel import converters
+
+
+if __name__ == "__main__":
```

Comment:
Interesting. Would you like this done as part of this PR, or should we open an issue for it?